implementations/rust/ockam/ockam_core/src/routing/route.rs

```diff
@@ -252,6 +252,11 @@ impl Route {
     pub fn is_local(&self) -> bool {
         self.iter().all(|a| a.is_local())
     }
+
+    /// Return inner VecDeque
```

Comment:
This function is not used.

Reply:
I that a problem?

Reply:
That's more code to inspect when trying to understand what's going on. But I'd rather not expose implementation details which you seem to like, seeing your comment above :-).